ramalama

```diff
@@ -183,14 +202,26 @@ def select_container_manager():
 
 
 def main():
+    conman = select_container_manager()
+    ramalama_store = get_ramalama_store()
+
+    if conman:
+        args = sys.argv
+        args.pop(0)
+        conman_args = [conman, "run", "--rm", "-it", "--security-opt=label=disable", f"-v{ramalama_store}:/var/lib/ramalama", f"-v{os.path.expanduser('~')}:{os.path.expanduser('~')}", "-v/tmp:/tmp",
+                       f"-v{__file__}:{__file__}", "quay.io/ramalama/ramalama:latest", __file__] + args
+        os.execvp(conman, conman_args)
+
     if len(sys.argv) < 2:
         usage()
 
-    ramalama_store = get_ramalama_store()
     command = sys.argv[1]
     if command == "pull" and len(sys.argv) > 2:
-        pull_cli(ramalama_store + "/repos/ollama",
-                 ramalama_store + "/models/ollama", sys.argv[2])
+        pull_cli(ramalama_store, sys.argv[2])
```

Comment:
switch command {
case "pull":
case "run":
}

Reply:
Do you have commands that handle 2 options?

Reply:
Yeah there will be:

ramalama list/ls

as an example that lists all models downloaded and available is coming.

Will change to switch, apparently they call it match in python for some reason

Reply:
How would we feel about going back to if/else if/else? The python switch equivalent in Python 3.10 (2021) and macOS build is failing here, I'm guessing the python3 version on macOS is older

Reply:
Going back to if, else, etc. fixed it anyway